Traceback (most recent call last):
  File "C:\Users\hp\Documents\code\Py\actual\Ai\inter_boot_2024\semseg\exp\interboot\pspnet50\test.py", line 254, in <module>
    main()
  File "C:\Users\hp\Documents\code\Py\actual\Ai\inter_boot_2024\semseg\exp\interboot\pspnet50\test.py", line 70, in main
    check(args)
  File "C:\Users\hp\Documents\code\Py\actual\Ai\inter_boot_2024\semseg\exp\interboot\pspnet50\test.py", line 48, in check
    assert (args.train_h - 1) % 8 == 0 and (args.train_w - 1) % 8 == 0
AssertionError
